table/table_builder.h

```diff
@@ -111,6 +111,7 @@ struct TableBuilderOptions : public TablePropertiesCollectorFactory::Context {
       bool _is_bottommost = false,
       TableFileCreationReason _reason = TableFileCreationReason::kMisc,
       const int64_t _oldest_key_time = 0,
+      const int64_t _newest_key_time = kUnknownNewestKeyTime,
```

Comment:
Can we make this a required parameter? It helps to find and catch places where we did not set this field.

Reply:
fair point. will do